dev_notes.txt

```diff
@@ -1,8 +1,13 @@
 # SSH'ing into the movos
 Make sure you are connected to the robohub wifi
-movo1 can be reached by `ssh robohub@129.97.70.109`
-movo2 can be reached by `ssh robohub@129.97.71.113`
+movo1 can be reached by `ssh robohub@movo1`
```

Comment:
can add that the IPs are 1 -> 129.97.71.49 and 2 -> 129.97.71.50

Reply:
im not sure those are the actual IPs. i think theyve changed since i wrote that. the alias is right tho

Reply:
Those are the IP's that Ben put for the aliases. They are in confluence in case we forget under "Movo"